{"trial_id": "000", "hyperparameters": {"space": [{"class_name": "Boolean", "config": {"name": "timeseries_block_1/rnn_block_1/bidirectional", "default": true, "conditions": []}}, {"class_name": "Choice", "config": {"name": "timeseries_block_1/rnn_block_1/layer_type", "default": "lstm", "conditions": [], "values": ["gru", "lstm"], "ordered": false}}, {"class_name": "Choice", "config": {"name": "timeseries_block_1/rnn_block_1/num_layers", "default": 2, "conditions": [], "values": [1, 2, 3], "ordered": true}}, {"class_name": "Choice", "config": {"name": "regression_head_1/dropout", "default": 0, "conditions": [], "values": [0.0, 0.25, 0.5], "ordered": true}}, {"class_name": "Choice", "config": {"name": "optimizer", "default": "adam", "conditions": [], "values": ["adam", "sgd", "adam_weight_decay"], "ordered": false}}, {"class_name": "Choice", "config": {"name": "learning_rate", "default": 0.001, "conditions": [], "values": [0.1, 0.01, 0.001, 0.0001, 2e-05, 1e-05], "ordered": true}}], "values": {"timeseries_block_1/rnn_block_1/bidirectional": true, "timeseries_block_1/rnn_block_1/layer_type": "lstm", "timeseries_block_1/rnn_block_1/num_layers": 2, "regression_head_1/dropout": 0, "optimizer": "adam", "learning_rate": 0.001}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"C:\\Users\\zcane\\miniconda3\\envs\\tf\\lib\\site-packages\\keras_tuner\\engine\\base_tuner.py\", line 250, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"C:\\Users\\zcane\\miniconda3\\envs\\tf\\lib\\site-packages\\keras_tuner\\engine\\base_tuner.py\", line 237, in _run_and_update_trial\n    self.oracle.update_trial(\n  File \"C:\\Users\\zcane\\miniconda3\\envs\\tf\\lib\\site-packages\\keras_tuner\\engine\\oracle.py\", line 105, in wrapped_func\n    ret_val = func(*args, **kwargs)\n  File \"C:\\Users\\zcane\\miniconda3\\envs\\tf\\lib\\site-packages\\keras_tuner\\engine\\oracle.py\", line 360, in update_trial\n    self._check_objective_found(metrics)\n  File \"C:\\Users\\zcane\\miniconda3\\envs\\tf\\lib\\site-packages\\keras_tuner\\engine\\oracle.py\", line 614, in _check_objective_found\n    raise ValueError(\nValueError: Objective value missing in metrics reported to the Oracle, expected: ['val_loss'], found: dict_keys([])\n"}